build.sbt

```diff
@@ -62,7 +85,18 @@ lazy val core = (project in file("atum"))
     (Compile / compile) := ((Compile / compile) dependsOn printSparkScalaVersion).value, // printSparkScalaVersion is run with compile
     mergeStrategy,
     populateBuildInfoTemplate // to get correct replacements for ${project.artifactId} and ${project.version} in atum_build.properties,
-  ).dependsOn(model)
+  )
+  .settings(jacocoCoreReportSettings)
+  .dependsOn(model)
+
+lazy val jacocos3SDKExtReportSettings = Seq(
+  jacocoReportSettings := JacocoReportSettings(
+    s"atum:s3sdkExtension Jacoco Report",
+    None,
+    JacocoThresholds(),
+    Seq(JacocoReportFormats.HTML, JacocoReportFormats.XML),
+    "utf-8")
```

Comment:
I am no expert in SBT, but I would assume that common `JacocoReportSettings` objects could defined for all usages
```
lazy val commonJacocoReportSettings := JacocoReportSettings(
  formats = Seq(JacocoReportFormats.HTML, JacocoReportFormats.XML) // other options are default
)
```

and then reused
```scala
commonJacocoReportSettings.withTitle("atum:s3sdkExtension Jacoco Report")
```

Looking at the [source code](https://github.com/sbt/sbt-jacoco/blob/main/src/main/scala/com/github/sbt/jacoco/report/JacocoReportSettings.scala), this should be possible. 

This should be, obviously, applicable elsewhere

Reply:
Resolved by commit 5789506e16af458426c5de6e9b110a2315408a2c.